Update to use Cache Ephemeral disk

dsrivastavv opened the pull request:
ResourceDisk size can be limited to less than marketplace image size leading to test failures

LiliDeng reviewed (changes requested):
need further discussion for this change

Commit:
Update to use Cache Ephemeral disk

ResourceDisk size can be limited to less than marketplace image size leading to test failures
